taxcalc/calculate.py

```diff
@@ -227,6 +228,46 @@ def zeroarray(self, variable_name):
         """
         setattr(self.records, variable_name, np.zeros(self.array_len))
 
+    @property
+    def array_len(self):
+        """
+        Length of arrays in embedded Records object.
+        """
+        return self.records.array_length
+
+    def getrecords(self):
```

Comment:
@martinholmer out of curiosity, why aren't you using an underscore in the method names `getrecords` and `setrecords`?  To me, the pattern indicates that this name should be `get_records` just like we have `set_records_current_year` below?

Reply:
Never mind, I see that you used `getarray` and `setarray`.  These names make more since now.

Reply:
Could be either way, @hdoupe, but I'm leaning toward short method names to ease the transition away from directly accessing a Records object embedded in a Calculator object.

Reply:
Both ways make sense to me.  I lean towards using the underscore.  This falls more into line with the Python variable naming convention.  Plus, I don't think adding an extra character costs very much.